node/test/hyperbahn/advertise.js

```diff
@@ -88,7 +88,7 @@ function runTests(HyperbahnCluster) {
         var client = new HyperbahnClient({
             serviceName: 'hello-bob',
             callerName: 'hello-bob-test',
-            hostPortFile: hostPortFile,
+            hostPortList: hostPortFile,
```

Comment:
this change is to simplify the constructor interface by reducing two parameters to one:
if (Array.isArray(hostPortList)) {
use it
} else if (typeof hostPortList is string) {
read and parse json file
} else {
throw error
}

As a user, I can get hostPort list from config or command line, it can be a file path or an array of strings. I just want to pass it into constructor, without worrying about which parameter to use.